nautobot/dcim/templates/dcim/device/wireless.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'dcim/device/base.html' %}
+{% load helpers %}
+{% load static %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-6">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Access Point Group</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Name</td>
+                        <td>
+                            {{ object.access_point_group|hyperlinked_object }}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Controller</td>
+                        <td>{{ object.access_point_group.controller|hyperlinked_object }}</td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-md-12">
+            <form method="post">
+                {% csrf_token %}
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <strong>Wireless Networks</strong>
+                        <div class="pull-right noprint">
+                            {% if request.user.is_authenticated %}
+                                {% table_config_button_small wireless_networks_table %}
```

Comment:
Same question on these table config cases - I ask because we've seen issues with getting them to work properly, especially with multiple tables in a view.

Reply:
I tested these in both Firefox and Safari, they both work. You can configure one and then the other, and they keep their settings between them.